### PR TITLE
🐛 Fix wrong offset Menu

### LINKED ISSUE #1212: Wrong offset on menu

### PR DESCRIPTION
**Pull request by pomfrida**
Resolves #1212

### CODE REVIEW

File: libraries/core-react/src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -100,6 +100,7 @@ export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(function Menu(
     containerRef.current,
     null,
     placement,
+    0,
```
Comment (mimarz) [inline libraries/core-react/src/components/Menu/Menu.tsx:103]: `0` looks very "tight" did you check with @equinor/eds-ux that this was ok value? I think value was more like 2 or 4px before... 🤔

**Review by mimarz — CHANGES_REQUESTED** [review]

File: libraries/core-react/src/components/Menu/Menu.tsx (hunk 1 above)
Comment (pomfrida) [reply libraries/core-react/src/components/Menu/Menu.tsx:103]: I used the context menu example [here](https://www.figma.com/file/luNobpDIWTuQ9rhyovGLBI/Desktop?node-id=977%3A27105) and it's 0px from anchor to menu

Comment (mimarz) [reply libraries/core-react/src/components/Menu/Menu.tsx:103]: 4px 
image

vs 

2px 
image

vs 

0px 
image

Comment (mimarz) [reply libraries/core-react/src/components/Menu/Menu.tsx:103]: I would say maybe `2px`? What do you guys think @equinor/eds-ux ?

Comment (BirteThornquist) [reply libraries/core-react/src/components/Menu/Menu.tsx:103]: @mimarz @pomfrida I think it shall be 4px space between the button and the drop menu. I guess the clickbound take care of that?

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**